### DISCUSSION TITLE
Rule only for files named package.json

### DISCUSSION THREAD

**Discussion by akmurat-h** [typescript-eslint/typescript-eslint#7150] [Technical Discussions]
### Body

Please tell me how to write a rule that will check only package.json files

### Additional Info

Any additional info...

### Before you submit your discussion, please confirm the following. If any of these required steps are not taken, we may not be able to review your RFC. Help us to help you!

- [X] I have [read the discussions guidelines](https://typescript-eslint.io/contributing/discussions) and per those guidelines, this fits a discussion category, _not_ a help request or standard issue.
- [X] I have [searched for related discussions](https://github.com/typescript-eslint/typescript-eslint/discussions) and [searched for related issues](https://github.com/typescript-eslint/typescript-eslint/issues) and found none that match my proposal.
- [X] I have [read the FAQ](https://typescript-eslint.io/linting/troubleshooting) and my problem is not listed.

**Comment by bradzacher** [comment]
As per the [discussion guidelines you ticked saying you read](https://typescript-eslint.io/contributing/discussions/). We do not use discussions as a support forum. Follow the link to find better places to ask your question!
